src/components/Header.tsx

```diff
@@ -132,6 +133,22 @@ const Header = () => {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, []);
 
+	useEffect(() => {
+  			if (!user) { return; }
+
+  			const isAdmin = user.isAdmin || user.isOrgAdmin;
+			// un comment the below line after the test
+  			// const notLocalhost = window.location.hostname !== "localhost";
+			// for testing purposes on the local host
```

Comment:
```suggestion
  			const notLocalhost = window.location.hostname !== "localhost";
```

I would also suggest flipping the logic here and making it this:
```suggestion
  			const isLocalhost = window.location.hostname == "localhost";
```
such that the if stanza on :148 reads `if (isAdmin && dismissedLongEnough && !isLocalhost) {`, which (IMO) reads cleaner

Reply:
Hi,
Done 🙂